.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
I have mixed feelings about this. imho this is something that should be included in a persons git ignore, but am not totally opposed to it. Would you be open to sending this in it's own pr and rebasing it out of this branch?